cgi-bin/DW/Controller/Support/Index.pm

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Support::Index
+#
+# This controller is for the Support Index page.
+#
+# Authors:
+#      hotlevel4 <hotlevel4@hotmail.com>
+#
+# Copyright (c) 2016 by Dreamwidth Studios, LLC.
+#
+# This is based on code originally implemented on LiveJournal.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Support::Index;
+
+use strict;
+use warnings;
+
+use DW::Controller;
+use DW::Routing;
+use DW::Template;
+
+DW::Routing->register_string( '/support/', \&index_handler, app => 1 );
+DW::Routing->register_string( '/support/index', \&index_handler, app => 1 );
+
+sub index_handler {
+    my $r = DW::Request->get;
+
+    my ( $ok, $rv ) = controller( anonymous => 1, form_auth => 1 );
+    return $rv unless $ok;
+
+    my $remote = $rv->{remote};
+    my $user;
+    my $user_url;
+
+    my $vars = {};
+    
+    my $currentproblems = LJ::load_include( "support-currentproblems" );
+    LJ::CleanHTML::clean_event( \$currentproblems, {} );    
+    $vars->{currentproblems} = $currentproblems;
+    
+    my $dbr = LJ::get_db_reader(); 
+    my $sth;
```

Comment:
I would move these two lines down to where `$sth` is assigned below; they are out of context here. I would also do `my $sth = $dbr->prepare(...);` -- I see no reason to separate the scope declaration from the assignment in this case.